sqldelight-idea-plugin/src/test/kotlin/app/cash/sqldelight/intellij/annotations/OptimisticLockAnnotatorTest.kt

```diff
@@ -63,7 +63,7 @@ class OptimisticLockValidatorTest : SqlDelightProjectTestCase() {
         |);
         |
         |updateText:
-        |<error descr="This query updates a table with an optimistic lock but does not correctly use the lock.">UPDATE test
+        |<error descr="This statement is missing an optimistic lock in it's SET clause.">UPDATE test
```

Comment:
Should this read "_the_ optimistic lock"? Can there be multiple? Can you only use one if there are? I'm out of my depth...

Reply:
You can definitely only use one...which is a good thing we should also print an error for